ListingService/Sources/SubredditSections.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public enum SubredditSections: Int {
+    case redditFeeds = 0
+    case defaultFeeds = 1
+    
+    public var title: String {
+        switch self {
+        case .redditFeeds:
+            return "Reddit Feeds"
+        case .defaultFeeds:
+            return "Default Feeds"
+        }
+    }
+    public static var allSections: [SubredditSections] {
+        return [.redditFeeds , .defaultFeeds]
+    }
+}
```

Comment:
Are we going to use this enum again, is a separate VC / datasource?

If not, I wonder if it would be better suited, placed within `SubredditsViewController.swift` perhaps in the `extension SubredditsViewController: UITableViewDataSource, UITableViewDelegate` as its related?

Reply:
No it's only used here. I will make the changes you recommended.